src/autopas/containers/linkedCells/LinkedCellsReferences.h

```diff
@@ -270,6 +270,40 @@ class LinkedCellsReferences : public CellBasedParticleContainer<ReferenceParticl
                                                                            nullptr));
   }
 
+  /**
+   * @copydoc LinkedCells::forEach()
```

Comment:
in this class, the same comments apply as for LinkedCells. Maybe a shared superclass is appropriate?

Reply:
Leave for now, reconsider after integrating kokkos.